docs/cli/cli-reference.md

```diff
@@ -0,0 +1,17 @@
+<<<<<<< HEAD
```

Comment:
Looks like some github diff stuff snuck in here... Would be good to remove it before merging.